pkg/utils/kong/fake_admin_api.go

```diff
@@ -12,6 +12,10 @@ import (
 // FakeAdminAPIServer - Public Types
 // -----------------------------------------------------------------------------
 
+var (
```

Comment:
why not const?

Reply:
changed in 2fb4ff0ee75a0a68bb0e367102747e84431b5028